webui/react/src/pages/ModelDetails.tsx

```diff
@@ -125,9 +123,9 @@ const ModelDetails: React.FC = () => {
     useCallback(async (editedDescription: string, versionId: number) => {
       try {
         await patchModelVersion({
-          body: { comment: editedDescription, id },
-          modelId: id,
-          versionId: versionId,
+          body: { comment: editedDescription, id: versionId },
```

Comment:
Unless you changed this on the backend, the `id` field in body in `patchModelVersion` is expecting the model ID, not the version ID.

Reply:
Thanks!  I got turned around on this. This should be fixed in any patchModelVersion calls with the latest commit